ticket_migrate.py

```diff
@@ -0,0 +1,829 @@
+#!/usr/bin/env python3
+# Migrate Trac tickets to GitHub.
+import datetime
+import pprint
+import sqlite3
+import sys
+from collections import deque
+from time import sleep
+from typing import Union
+
+import requests
+
+try:
+    import config
+except ModuleNotFoundError:
+    # In the tests, we monkeypatch this module.
+    config = None
+
+from trac2down import convert
```

Comment:
Trac2down is thankfully licensed as LGPL, so linking from non-GPL code is allowed.